src/components/SummaryCard/SummaryCardContainer/SummaryCardContainer.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file Summary card container.
+ * @copyright IBM Security 2019
+ */
+
+import React, { useState } from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+
+import { namespace as summaryCardNamespace } from '../SummaryCard';
+import { namespace as summaryCardSelectNamespace } from '../SummaryCardSelect/SummaryCardSelect';
+
+const getSummaryCard = (summaryCards, selectedId) =>
+  summaryCards.filter(({ id }) => id === selectedId);
+
+const resetSelectedSummaryCards = state => state([]);
+
+const SummaryCardContainer = ({ render, summaryCards }) => {
```

Comment:
Could we start to use a `function` declaration for components constructed in this way, that use hooks? 

It matches the format of the React docs (https://reactjs.org/docs/hooks-overview.html) & separately it was brought up to me in a recent PR for Carbon core. It sounds like a direction they want to go, in terms of how their components are written/constructed, so perhaps we can start going in the same direction as well?

Reply:
That makese sense!